boot.sh

```diff
@@ -92,6 +112,10 @@ fi
 if [ -n "${kubeadm_data}" ] ; then
     metadata="${metadata:+$metadata, }\"kubeadm\": { \"entries\": { ${kubeadm_data} } }"
 fi
+
+kubelet_config_escaped=$(echo "${KUBELET_CONFIG//$'\n'/}" | sed "s/\"/\\\\\"/g" | sed "s/ //g")
```

Comment:
I wonder if we can rely on `jq` here (any reason why not?) if we can then this can become the much more obviously correct:
```
kubelet_config_escaped=$(echo "${KUBELET_CONFIG//$'\n'/}" | jq tostring)
```
(possibly just `echo ${KUBELET_CONFIG` too since I'm not sure that `jq` minds about a trailing newline either way)

Reply:
Yeah, with `jq` this would be much simple but I didn't want to add `jq` dependency. You think it's ok to add it?

Reply:
Actually after moving the logic of generating the config to `kubelet.sh`, I don't need to format it as a string.